src/views/Wiki.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <main class="wiki container">
+    <aside>
+      <h1>Wiki</h1>
+      <sidebar />
```

Comment:
```suggestion
      <sidebar />
      <br />
```

The last link is already pretty far below.
![grafik](https://user-images.githubusercontent.com/43016900/78603832-105a5780-7859-11ea-9a59-62f85cdfb437.png)

Maybe consider adding a `<br />`? Looks like this then:
![grafik](https://user-images.githubusercontent.com/43016900/78603956-4a2b5e00-7859-11ea-8575-9921058e2778.png)

Reply:
Hmm doesn't display like that for me. Is that Firefox?

Reply:
Displays like that for me too on Firefox!

Reply:
Classic Firefox, at least it's not Safari 😝